src/NIDAQ.jl

```diff
@@ -33,14 +33,17 @@ export stop, clear
 
 # channels
 export analog_input, analog_output, digital_input, digital_output
+export acceleration_input
 export count_edges, quadrature_input, line_to_line, generate_pulses
 
 # properties
-export devices, channel_type, getproperties, setproperty!
-export analog_input_ranges, analog_output_ranges
-export analog_input_channels, analog_output_channels
-export digital_input_channels, digital_output_channels
-export counter_input_channels, counter_output_channels
+export devices, channel_type, getproperties #, setproperty!
+export analog_input_ranges,          analog_output_ranges
+export analog_volt_input_ranges,     analog_volt_output_ranges
```

Comment:
i'd suggest `analog_voltage_input_ranges` instead of shortening it to `analog_volt_input_ranges` since `current` is spelled completely out.

Reply:
That´s right, I will do this and commit

Reply:
great!  make sure to *not* commit your changes in #34  to this PR when doing so.